compiler/frontend/beam/src/main/java/edu/snu/nemo/compiler/frontend/beam/PipelineTranslator.java

```diff
@@ -326,6 +388,10 @@ private TranslationContext(final CompositeTransformVertex pipeline,
       this.pipelineOptions = pipelineOptions;
     }
 
+    public CompositeTransformVertex getPipeline() {
```

Comment:
Do we really need this 'public' method? PipelineTranslator is a function that takes a pipeline and emits the corresponding IR. PipelineTranslator users do not have to rely on this method to obtian the pipeline, becuase it already has the pipeline.